server/app.js

```diff
@@ -20,57 +27,58 @@ const storage = multer.diskStorage({
     cb(null, file.originalname);
   },
 });
+
 const upload = multer({ storage: storage });
 
 app.use(logger('dev'))
 app.use(express.json())
 app.use(express.urlencoded({extended: true}))
 app.use(cors())
+
+
 app.use('/api/qa', router)
 
```

Comment:
I basically uprooted my answers post controller route and did the logic in the server lol. Probably will reformat later if I have time. All other api's are still in my router file.